gtk/src/auto/widget.rs

```diff
@@ -91,7 +91,7 @@ pub trait WidgetExt: 'static {
 
     fn create_pango_layout(&self, text: Option<&str>) -> pango::Layout;
 
-    //fn destroyed<P: IsA<Widget>>(&self, widget_pointer: &P);
+    //fn destroyed<P: IsA<Widget>>(&self, widget_pointer: P);
```

Comment:
Why did this change? :)

Reply:
I think that's https://github.com/gtk-rs/gir/pull/991/commits/08c5ff7f6c390e15638cd7023917c55222fccc73.

Reply:
Hm maybe something broken with commented out code. It doesn't have any effect on any of the actually generated code :man_shrugging: Let's ignore that then or what do you think? Also @GuillaumeGomez?

Reply:
> It doesn't have any effect on any of the actually generated code

It changes all enum functions to take `self` by move/copy instead of `&self`.

Reply:
Sure, I mean it has no effect on previously generated code (i.e. it doesn't break anything) :)

Reply:
I wonder why this function is not generated; is it the double pointer?

When someone looks into it I guess this is going to be a `&mut *mut P`, right?

Reply:
Yes that's the reason here